test/godeps/lxd-agent.list

```diff
@@ -13,7 +13,6 @@ github.com/canonical/lxd/lxd/metrics
 github.com/canonical/lxd/lxd/operations
 github.com/canonical/lxd/lxd/request
 github.com/canonical/lxd/lxd/response
-github.com/canonical/lxd/lxd/state
```

Comment:
So when using the build tags to list dependencies, this refactor only removes a single package! I guess that explains why the binary size didn't get smaller - though it doesn't explain why it got larger.